gravitee-apim-portal-webui-next/src/components/drawer/drawer.component.scss

```
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+:host {
+  .drawer-container {
+    &__content {
+      transition: all 300ms;
+
+      &--hidden {
+        display: none;
+      }
+    }
+  }
+}
```

You don't need to put your classes in `:host {}`. They can go directly in the stylesheet 👍 
```suggestion
  .drawer-container {
    &__content {
      transition: all 300ms;

      &--hidden {
        display: none;
      }
    }
  }
```